packages/openapi-generator/src/generator-utils.ts

```diff
@@ -0,0 +1,50 @@
+import { promises } from 'fs';
+import { resolve } from 'path';
+import { ErrorWithCause } from '@sap-cloud-sdk/util';
+import {
+  GeneratorOptions,
+  ParsedGeneratorOptions,
+  generatorFlags
+} from './options';
+const { readFile, lstat } = promises;
+
+/**
+ * Parses a given path to a config file or directory and returns its content
+ * @param configPath path to a config file or a directory containing a config.json
+ * @returns Options to configure generation
+ */
+export async function parseOptionsFromConfig(
```

Comment:
Actually, why isn't this in the `generator-options.ts`? I think it would fit there better ;)

Reply:
Agreed, moved it over to generator-options 👍